DXRModules/DeusEx/Classes/DXRSkills.uc

```diff
@@ -174,6 +174,21 @@ simulated function RandoSkillLevelValues(Skill a)
 }
 
 simulated function string DescriptionLevel(Actor act, int i, out string word, out float val, float defaultval)
+{
+    local string shortDisplay;
+    return DescriptionLevelExtended(act, i, word, val, defaultval, shortDisplay);
+}
+
+simulated function string DescriptionLevelShort(Actor act, int i, out float val)
```

Comment:
these changes should maybe be in the base class DXRBase, or at least they will need to be moved for Astro's DXMP-style aug upgrade menu